src/tools/linter-rules/index.md

```diff
@@ -146,5 +146,5 @@ These rules identify possible issues around
 
 {% include linter-rules-section.md type="pub" %}
 
-[Enabling and disabling linter rules]: /guides/language/analysis-options#enabling-linter-rules
+[Enabling and disabling linter rules]: /tools/analysis#enabling-linter-rules
```

Comment:
I don't think this link is used anywhere on the page (it's already linked on 66 and only used in one place), can probably delete it